tests/posit/connect/conftest.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+from posit.connect import config
+
+
+def pytest_configure():
+    c = config.Config()
+    c.api_key = "12345"
+    c.url = "https://connect.example/__api__"
```

Comment:
This runs at the beginning of the test suite. Therefore, tests can assume these values when creating mocks or whatever else.